README.md

```diff
@@ -30,10 +30,22 @@ You must have a Cloudflare account and your domain must be configured to point t
 - `Hostname`: the full subdomain and hostname of the record you want to update (e.g. `subdomain.mydomain.com`, `mydomain.com` for root domain)
 - `Username`: the domain name containing the record (e.g. `mydomain.com`)
 - `Password`: the Cloudflare API Token you created earlier
-- `Server`: the Cloudflare Worker route `<worker-name>.<worker-subdomain>.workers.dev/update?ip=%i&hostname=%h`. 
+- `Server`: the Cloudflare Worker route `<worker-name>.<worker-subdomain>.workers.dev/update?ip=%i&hostname=%h`
+  - On UniFi devices older than the UDM the `Server` value should be configured with no URL path: `<worker-name>.<worker-subdomain>.workers.dev`.
+  - Many UniFi devices do not handle the `https://` prefix in the URL for the `Server`
+    value. You should remove it, using `<worker-name>.<worker-subdomain>.workers.dev`
+    instead of `https://<worker-name>.<worker-subdomain>.workers.dev`.
+
+To test the configuration and force an update:
+
+1. Ssh into your UniFi device
+2. Run `ps aux | grep inadyn`
+3. Note the configuration file path referenced by the running inadyn process e.g.
+   `/run/ddns-eth4-inadyn.conf`.
+4. Run `inadyn -n -1 --force -f <config-path>` e.g. `inadyn -n -1 --force -f /run/ddns-eth4-inadyn.conf`
+
+Also check `/var/log/messages` for error messages related to `inadyn`.
```

Comment:
For my USG 3P/USG Pro 4, I needed to look for errors from `ddclient` (`tail -f -n1000 /var/log/messages | grep 'ddclient'`)

Reply:
@huwcbjones want to send another PR? It's interesting that different models use different dynamic DNS clients.